core/src/main/resources/reference.conf

```diff
@@ -392,7 +392,25 @@ services {
   MetadataService {
     class = "cromwell.services.metadata.impl.MetadataServiceActor"
     # Set this value to "Inf" to turn off metadata summary refresh.  The default value is currently "2 seconds".
+    # FIXME: move
```

Comment:
Phew I'm not going crazy. I thought I had fixed this and the changelog previously, I just hadn't pushed it.